package.json

```diff
@@ -63,7 +63,7 @@
     "react-select": "^3.1.0",
     "react-sortablejs": "^6.0.0",
     "react-youtube": "^7.12.0",
-    "socket.io-client": "^2.2.0",
+    "socket.io-client": "^4.1.2",
```

Comment:
This introduced a regression for older browsers —such as CatQuest's FF44 — either the newer version of this library or one of its dependencies isn't compiled to ES5.
I'll open a PR to fix it.

Reply:
Oh! Thanks!

Reply:
For posterity -> #1524